docs/content/contributing/_index.en.md

```diff
@@ -26,33 +26,48 @@ directory (if you want to build the image with the docker inside
 minikube, you'll need to set the proper environment variables with
 `eval $(minikube docker-env)`):
 
-```
-  # Get dependencies
-  $ glide install
+Install dependencies with glide:
 
-  # Build fission server and an image
-  $ pushd fission-bundle
-  $ ./build.sh
-```
+{{< highlight bash >}}
+$ glide install
```

Comment:
Add `-v` flag to prevent some problem with nested dependencies problem.
`$ glide install -v`